src/test/java/com/epam/tc/hw4/TestStepsHomePage.java

```diff
@@ -0,0 +1,155 @@
+package com.epam.tc.hw4;
+
+import com.epam.tc.hw4.pages.HomePage;
+import io.qameta.allure.Step;
+import java.util.Arrays;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+
+
+public class TestStepsHomePage {
+    private HomePage homePage;
+    SoftAssertions softAssert = new SoftAssertions();
```

Comment:
Why are you using softAssertion?

Reply:
I use softassert so that if the test fails, the testing process continues and all other tests are also executed